### ISSUE TITLE
`requestContext.requestId` in lambda http event is missing

### ISSUE THREAD

**Issue by milenkovicm** [localstack/localstack#4818]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

There is an issue with api gateway and lambda integration (rust sdk). Deployed lambda will be invoked but event received will not have all required properties.

Local stack in use:

```
LocalStack version: 0.12.19.1
LocalStack Docker container id: 65146bae2247
LocalStack build date: 2021-10-29
LocalStack build git hash: 6fa54b3e
```

Event received from localstack api gateway does not contain  `"requestId": "xxxxxxxxxxx" ` in the `requestContext` object.

Example event:

```json
{
    "path": "/helloo",
    "headers": {
        "remote-addr": "172.17.0.1",
        "host": "localhost:4566",
        "user-agent": "curl/7.64.1",
        "accept": "*/*",
        "x-forwarded-for": "172.17.0.1, localhost:4566, 127.0.0.1, localhost:4566",
        "x-localstack-edge": "http://localhost:4566",
        "authorization": "AWS4-HMAC-SHA256 Credential=__internal_call__/20160623/us-east-1/apigateway/aws4_request, SignedHeaders=content-type;host;x-amz-date;x-amz-target, Signature=1234",
        "x-localstack-tgt-api": "apigateway",
        "__apigw_request_region__": "us-east-1"
    },
    "multiValueHeaders": {
        "remote-addr": [
            "172.17.0.1"
        ],
        "host": [
            "localhost:4566"
        ],
        "user-agent": [
            "curl/7.64.1"
        ],
        "accept": [
            "*/*"
        ],
        "x-forwarded-for": [
            "172.17.0.1, localhost:4566, 127.0.0.1, localhost:4566"
        ],
        "x-localstack-edge": [
            "http://localhost:4566"
        ],
        "authorization": [
            "AWS4-HMAC-SHA256 Credential=__internal_call__/20160623/us-east-1/apigateway/aws4_request, SignedHeaders=content-type;host;x-amz-date;x-amz-target, Signature=1234"
        ],
        "x-localstack-tgt-api": [
            "apigateway"
        ],
        "__apigw_request_region__": [
            "us-east-1"
        ]
    },
    "body": "",
    "isBase64Encoded": false,
    "httpMethod": "GET",
    "queryStringParameters": {
        "first_name": "xxxxxxxxxx"
    },
    "multiValueQueryStringParameters": {
        "first_name": [
            "xxxxxxxxxx"
        ]
    },
    "pathParameters": {
        "proxy": "helloo"
    },
    "resource": "/{proxy+}",
    "requestContext": {
        "path": "/test/helloo",
        "resourcePath": "/{proxy+}",
        "apiId": "o2pawlcr40",
        "domainPrefix": "o2pawlcr40",
        "domainName": "o2pawlcr40.execute-api.localhost.localstack.cloud",
        "accountId": "000000000000",
        "resourceId": "61mg7wiqtz",
        "stage": "test",
        "identity": {
            "accountId": "000000000000",
            "sourceIp": "127.0.0.1",
            "userAgent": "curl/7.64.1"
        },
        "httpMethod": "GET",
        "protocol": "HTTP/1.1",
        "requestTime": "2021-10-29T23:57:19.839Z",
        "requestTimeEpoch": 1635551839839
    },
    "stageVariables": {}
}
```

Also, `requestTime` and `requestTimeEpoch` are not properites of object (api gateway V2 has `time` and `timeEpoch`).
I could not find schema for given event, to confirm schema requirements.

Links:

- [Test event used in rust aws api](https://github.com/awslabs/aws-lambda-rust-runtime/blob/master/lambda-http/tests/data/apigw_proxy_request.json)
- [Struct object representing event](https://github.com/awslabs/aws-lambda-rust-runtime/blob/40e2658804a75377b96a199a18f195122828c38d/lambda-http/src/request.rs#L70)

### Expected Behavior

lambda http event should contain 

```json
{
    "path": "/helloo",
    "headers": {
        "remote-addr": "172.17.0.1",
        "host": "localhost:4566",
        "user-agent": "curl/7.64.1",
        "accept": "*/*",
        "x-forwarded-for": "172.17.0.1, localhost:4566, 127.0.0.1, localhost:4566",
        "x-localstack-edge": "http://localhost:4566",
        "authorization": "AWS4-HMAC-SHA256 Credential=__internal_call__/20160623/us-east-1/apigateway/aws4_request, SignedHeaders=content-type;host;x-amz-date;x-amz-target, Signature=1234",
        "x-localstack-tgt-api": "apigateway",
        "__apigw_request_region__": "us-east-1"
    },
    "multiValueHeaders": {
        "remote-addr": [
            "172.17.0.1"
        ],
        "host": [
            "localhost:4566"
        ],
        "user-agent": [
            "curl/7.64.1"
        ],
        "accept": [
            "*/*"
        ],
        "x-forwarded-for": [
            "172.17.0.1, localhost:4566, 127.0.0.1, localhost:4566"
        ],
        "x-localstack-edge": [
            "http://localhost:4566"
        ],
        "authorization": [
            "AWS4-HMAC-SHA256 Credential=__internal_call__/20160623/us-east-1/apigateway/aws4_request, SignedHeaders=content-type;host;x-amz-date;x-amz-target, Signature=1234"
        ],
        "x-localstack-tgt-api": [
            "apigateway"
        ],
        "__apigw_request_region__": [
            "us-east-1"
        ]
    },
    "body": "",
    "isBase64Encoded": false,
    "httpMethod": "GET",
    "queryStringParameters": {
        "first_name": "xxxxxxxxxx"
    },
    "multiValueQueryStringParameters": {
        "first_name": [
            "xxxxxxxxxx"
        ]
    },
    "pathParameters": {
        "proxy": "helloo"
    },
    "resource": "/{proxy+}",
    "requestContext": {
        "path": "/test/helloo",
        "resourcePath": "/{proxy+}",
        "apiId": "o2pawlcr40",
        "domainPrefix": "o2pawlcr40",
        "domainName": "o2pawlcr40.execute-api.localhost.localstack.cloud",
        "accountId": "000000000000",
        "resourceId": "61mg7wiqtz",
        "stage": "test",
        "identity": {
            "accountId": "000000000000",
            "sourceIp": "127.0.0.1",
            "userAgent": "curl/7.64.1"
        },
        "httpMethod": "GET",
        "protocol": "HTTP/1.1",
        "requestTime": "2021-10-29T23:57:19.839Z",
        "requestTimeEpoch": 1635551839839, 
        "requestId": "xxxxxxxxxxx"
    },
    "stageVariables": {}
}
```

`"requestId": "xxxxxxxxxxx"` should be included in `requestContext`

### How are you starting LocalStack?

With a `docker run` command

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```bash
docker run -e "SERVICES=lambda,apigateway" \
    -e LAMBDA_EXECUTOR=docker \
    -e LAMBDA_REMOTE_DOCKER=true \
    -e LAMBDA_REMOVE_CONTAINERS=true \
    -e DOCKER_HOST=unix:///var/run/docker.sock --rm \
    -e DATA_DIR=/tmp/localstack/data \
    -p 4566:4566 -p 4571:4571 \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v /Users/xxxxxxxx/TMP/localstack:/tmp/localstack \
    localstack/localstack
```

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

can be provided later if needed

### Environment

```markdown
- OS: MacOS
- LocalStack: latest
```

**Comment by whummer** [comment]
Thanks for reporting @milenkovicm - this should be fixed in #4843 . The `latest` Docker image is currently being built in our CI pipeline, and should become available on Docker Hub in ~40 minutes from now.. Thanks!

**Comment by milenkovicm** [comment]
Thanks @pinzon and @whummer it looks fixed now. Rust lambda works with api gateway!

Awesome work!
